src/amber/server/env.cr

```diff
@@ -0,0 +1,27 @@
+module Amber
+  module Env
+    ENVIRONMENTS = %w(development test production)
```

Comment:
These could be dynamically defined at compile time by looking at file names in `./config/environments`

Reply:
Do you have an example on how to do this? I got stuck...

Reply:
Yeah. Give me a second.

Reply:
There are 2 ways to do this.

### Eval
This way requires less files but isn't quite as fast for compile time.

`ENVIRONMENTS = {{ system("crystal eval \'puts Dir.glob(\"./config/environments/**\").map(&.split(/[\\/\\.]/)[-2]?)\'") }}`

### New File using Macro Run

This way is a bit faster as the compile can cache the compiled `environment_names.cr`

```cr
# environment_names.cr
puts Dir.glob("./config/environments/**").map(&.split(/[\/\.]/)[-2]?)
```

Then call `ENVIRONMENTS = {{run("./path/to/environment_names.cr")}}` in env.cr.